src/redux/settings/erc20Manager/tokens.js

```diff
@@ -93,6 +93,7 @@ export const formTokenLoadMetaData = async (token: TokenModel, dispatch) => {
 
   dispatch({type: TOKENS_FORM_FETCH, end: true})
 
+  // TODO @ipavlenko: check a list of existent tokens, not only ETH
```

Comment:
this check is provided by
`const symbolAddress = await managerDAO.getTokenAddressBySymbol(token.symbol())`
if symbolAddress is null, then there is no token with specified symbol in contract yet
otherwise specified symbol is already in use
so delete this TODO